docs/developer/meta/config-specs.md

```diff
@@ -141,6 +141,8 @@ It also respects a few extra fields under the `value` attribute of each option:
 
 Use the `--sync` flag of the [config validation command](../ddev/cli.md#ddev-validate-config) to render the example configuration files.
 
+**Note:** Python-based core checks contain a `conf.yaml.default` file that do not use the configuration specification files. These files must be updated manually.
```

Comment:
They do but you must define this: https://github.com/DataDog/integrations-core/blob/695587ee30d69acf27d266c32380ad1c1c794a20/disk/assets/configuration/spec.yaml#L4

Reply:
Ah thanks! I'll update this